### PR TITLE
Fix #11

### LINKED ISSUE #11: Incorrect testnet installation path for genesis.json and config.json
I've noticed that the installation path for genesis.json and config.json files for the testnet is incorrect.

Currently, the installation path for these files is set as https://files.deploy.aurora.dev/testnet-new-rpc/, which results in an error when trying to install them.

The correct installation path for the testnet files should be https://files.deploy.aurora.dev/testnet/.

Please update the installation path for these files so that users can install and use the testnet without any issues.

Current: Not available paths
https://files.deploy.aurora.dev/testnet-new-rpc/config.json
https://files.deploy.aurora.dev/testnet-new-rpc/genesis.json.gz

To be:
https://files.deploy.aurora.dev/testnet/config.json
https://files.deploy.aurora.dev/testnet/genesis.json.gz

### PR DESCRIPTION
**Pull request by nujabes403**
Fix #11

### CODE REVIEW

**Comment by nujabes403** [comment]
@akeyder

**Review by akeyder — APPROVED** [review]
LGTM

**Comment by nujabes403** [comment]
> Fix #11 

@tolga-coplu @spilin 
Could you review this code?

File: install.sh [hunk 1]
```diff
@@ -9,6 +9,15 @@ migrate_from=""
 use_snapshots=1
 download_workers=256
 
+# config.json, genesis.json install url
+if [ "$network" = "testnet" ]; then
+  config_install_url="https://files.deploy.aurora.dev/testnet/config.json"
```
Comment (Karkunow) [inline install.sh:14]: @nujabes403 Hi! This is the old config, and if you try this link – it won't work right now.
The correct config should be at:
https://files.deploy.aurora.dev/testnet-new-rpc/config.json

Comment (nujabes403) [reply install.sh:14]: Thanks. @Karkunow

**Comment by akeyder** [comment]
Hi @nujabes403, we decided to publish the config at the location expected in install script so this PR now obsolete, thanks for catching the problem!

**Comment by nujabes403** [comment]
@akeyder no problem :)

**Pull request closed without merging**